gson/src/main/java/com/google/gson/stream/JsonWriter.java

```diff
@@ -176,6 +176,7 @@
    */
   private String indent;
 
+  private boolean prettyPrinting = false;
```

Comment:
Is this field necessary? The `indent` field seems to cover this case.

Reply:
I've removed it.